tests/archive/tests_management_command.py

```diff
@@ -752,3 +752,23 @@ def test_suspend_command_setting(self, settings, suspended, caplog, snapshot):
         settings.SUSPEND_ANONYMIZE_PROFESSIONALS = suspended
         call_command("anonymize_professionals", wet_run=True)
         assert caplog.messages[0] == snapshot(name="suspend_anonymize_professionals_command_log")
+
+    @pytest.mark.parametrize("factory", [EmployerFactory, PrescriberFactory, LaborInspectorFactory])
+    def test_reset_notified_professional_dry_run(self, factory):
+        user = factory(joined_days_ago=DAYS_OF_INACTIVITY, notified_days_ago=1, last_login=timezone.now())
+        call_command("anonymize_professionals")
+
+        unmodified_user = User.objects.get()
+        assert user == unmodified_user
```

Comment:
ce test est censé vérifier que le champ n'a pas changé ?

Reply:
Oui, verifier que l'objet n'a pas été modifié en mode `wet_run`. As-tu une suggestion pour l'améliorer ou le rendre plus explicite ?

Reply:
https://github.com/django/django/blob/main/django/db/models/base.py#L615-L623

ça ne vérifie que la PK :)

Reply:
```
user.refresh_from_db()
assert user.upcoming_deletion_notified_at is None
```

Reply:
merci, je mets à jour

Reply:
je ne comprends pas le `is None`, puisque nous voulons vérifier que la date de notification n'est pas supprimée en mode `wet_run`.

Reply:
Ah, oui, `is not None` alors :see_no_evil: